pallets/hermes-governance-platform/src/lib.rs

```diff
@@ -46,21 +42,32 @@ pub struct HermesPollInfo<AccountId, Moment> {
     /// Poll end timestamp
     pub poll_end_timestamp: Moment,
     /// Poll title
-    pub title: String,
+    pub title: BoundedString<StringLimit>,
     /// Description
-    pub description: String,
+    pub description: BoundedString<StringLimit>,
     /// Creator Hermes withdrawn
     pub creator_hermes_withdrawn: bool,
+    /// Options
+    pub options: Vec<BoundedString<StringLimit>>,
```

Comment:
Maybe also use BoundedVec instead of Vec?

Reply:
Done.